tests/1.Mnemonics_test.ts

```diff
@@ -1,8 +1,8 @@
 /* eslint-env mocha */
-const assert = require('assert');
-const algosdk = require('../src/index');
-const nacl = require('../src/nacl/naclWrappers');
-const passphrase = require('../src/mnemonic/mnemonic');
+import assert from 'assert';
+import algosdk from '../src/index.js';
+import { randomBytes } from '../src/nacl/naclWrappers.js';
+import * as passphrase from '../src/mnemonic/mnemonic.js';
```

Comment:
What's the difference between lines 3 and 5?

And did you choose not to import `nacl` because you're only using one function?

Reply:
The difference is whether the module has a default export or not. In line 3, `../src/index.js` file has a default export (shown [here](https://github.com/algorand/js-algorand-sdk/blob/482cfe15553821dfcd357246e6b787b02f89fd2d/src/index.ts#L4)), while `../src/mnemonic/mnemonic.js` from line 5 does not, so we have to either import each definition explicitly, or use `* as X` to combine all the exports into one object.

And yes, since we only used one definition from `nacl`, I opted to just import that directly instead of couching it in an object.